Features/ContentTypeManagement/field_definitions_automatic_ordering.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Automatic ordering of field definitions when editing a content type
+  In order to manage content types
+  As an Administrator user
+  I need to be able to see field definitions sorted in a sensible, predictable order
+
+  Background:
+    Given I am logged in as admin on PlatformUI
+
+  ##
+  #
+  # Order by position
+  #
+  ##
+  @javascript @common
+  Scenario: Order field types of Article content type by position
+    Given an "Article" content type exists
+    And I am on the "Article" content type view page
+    When I edit the "Article" content type
+    Then I should see the following field definitions:
```

Comment:
I think that this list can now be shortened, with `clean` being the install used by Behat.